src/inverseRobot.js

```diff
@@ -4,10 +4,34 @@
  * @param {object} robot
  *
  * @return {object}
-*/
-
+ */
 function inverseRobot(robot) {
-  // write code here
+  const robotKeys = Object.values(robot);
+  const countObj = {};
+
+  for (let i = 0; i < robotKeys.length; i++) {
+    const elem = robotKeys[i];
+
+    if (!countObj[elem]) {
+      countObj[elem] = 1;
+    } else {
+      countObj[elem] += 1;
+    }
+  }
+
+  const robotValues = Object.values(countObj).some((num) => num > 1);
+
+  if (robotValues) {
+    return null;
+  }
+
+  const invertObj = Object.fromEntries(
```

Comment:
The current implementation only reverses the object entries and maps them, which does not guarantee the correct order of keys and values. Use a simpler approach with a new object and a 'for ... in' loop to swap keys and values.